uv.gyp

```diff
@@ -274,19 +274,44 @@
           },
         }],
         [ 'OS=="aix"', {
-          'sources': [ 'src/unix/aix.c' ],
+          'variables': {
+            'TrueOS': '<!(uname -s)',
+          },
+          'sources': [ 
+            'src/unix/aix-ibmi.c',
+          ],
           'defines': [
             '_ALL_SOURCE',
             '_XOPEN_SOURCE=500',
             '_LINUX_SOURCE_COMPAT',
             '_THREAD_SAFE',
-            'HAVE_SYS_AHAFS_EVPRODS_H',
           ],
-          'link_settings': {
-            'libraries': [
-              '-lperfstat',
-            ],
-          },
+          'conditions': [
+            [ '"<(TrueOS)"=="OS400"', {
+              'sources': [ 
+                'src/unix/ibmi.c',
+                'src/unix/posix-poll.c',
+                'src/unix/no-fsevents.c',
+                'src/unix/no-proctitle.c',
+              ],
+              'defines': [
+                '_PASE'
```

Comment:
I'm curious, isn't that set automatically by xlc?  (And gcc too, presumably?)

Reply:
gcc is the compiler we used and it would not set this.

Reply:
Given that there are more differences than commonalities in this section between aix and OS400, I think the os400 section should be separate from aix maybe!
```gyp
[ 'OS=="aix"', {
   ...
}],
[ 'OS=="os400"', {
   ...
}],
```

Reply:
No. Maybe in future we could build an IBM i gcc which can handle this. But for now, we use the AIX gcc which does not automatically add this macro.

Reply:
Can you define it as `_PASE=1` for consistency?